packages/fluttium/lib/src/tester.dart

```diff
@@ -23,7 +23,8 @@ class Tester {
 
   final Registry _registry;
 
-  SemanticsOwner get _semanticsOwner => _binding.pipelineOwner.semanticsOwner!;
+  SemanticsOwner? get _semanticsOwner =>
+      RendererBinding.instance.rootPipelineOwner.semanticsOwner;
```

Comment:
```suggestion
      _binding.rootPipelineOwner.semanticsOwner;
```

We should still use the `_binding` so we can test the tester!